### PR TITLE
Topic/tz

### PR DESCRIPTION
**Pull request by erikd-ambiata**
Monday morning yak shave. Trying to get relic building with ghc-7.10.

! @nhibberd

### CODE REVIEW

File: boris.toml [hunk 1]
```diff
@@ -8,5 +8,8 @@
 [build.validate]
```
Comment (nhibberd) [inline boris.toml]: This won't build anymore, needs to be updated with dist-validate

File: delorean.cabal [hunk 2]
```diff
@@ -19,8 +19,8 @@ library
                      , attoparsec                      >= 0.11       && < 0.14
                      , template-haskell
                      , text                            == 1.2.*
-                     , time                            >= 1.4        && < 1.6
-                     , tz                              <  0.1.1
+                     , time                            >= 1.4        && < 1.7
+                     , tz                              >=  0.1       && < 0.2
```
Comment (nhibberd) [inline delorean.cabal]: This won't compatible upstream given the previous range

File: boris.toml (hunk 1 above)
Comment (erikd-ambiata) [inline boris.toml]: Fixed!

File: delorean.cabal (hunk 2 above)
Comment (erikd-ambiata) [inline delorean.cabal]: Upstream? Deloren itself builds fine with tz 0.1.2.1.

Comment (nhibberd) [inline delorean.cabal]: Just going off the first example I looked up - https://github.com/ambiata/hydra/blob/master/hydra-daemon/hydra-daemon.cabal#L56.  Ideally this change should be compatible as to not force some mass rollout

Comment (jystic) [inline delorean.cabal]: Yeah, should be `>=  0.0       && < 0.2` really to be compatible with the old bounds that were there

Comment (erikd-ambiata) [inline delorean.cabal]: `>=  0.0 && < 0.2` it is then.

**Comment by nhibberd** [comment]
:+1:

**Pull request merged**